backend/presentation/views.py

```diff
@@ -0,0 +1,45 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from presentation.models import Presentation, Tag
+from presentation.permissions import IsAdminOrOwner, IsAdminOrReadOnly
+from presentation.serializers import (
+    InputPresentationSerializer,
+    OutputPresentationSerializer,
+    TagSerializer,
+)
+from presentation.services import PresentationService
+
+
+class PresentationViewSet(ModelViewSet):
+    queryset = Presentation.objects.all()
+    serializer_class = OutputPresentationSerializer
+    permission_classes = [IsAdminOrOwner]
+    services = PresentationService()
+
+    def create(self, request, *args, **kwargs):
+        serializer = InputPresentationSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        presentation = self.services.create_presentation(
+            presentation_data=serializer.validated_data
+        )
+        return Response(data=self.get_serializer(presentation).data, status=status.HTTP_201_CREATED)
```

Comment:
A little space for code to breath will improve readability
```suggestion
        )

        return Response(data=self.get_serializer(presentation).data, status=status.HTTP_201_CREATED)
```